lib/prerequisite.js

```diff
@@ -11,14 +11,15 @@ module.exports = (input, pkg, opts) => {
 	const tasks = [
 		{
 			title: 'Ping npm registry',
+			skip: () => pkg.private || typeof pkg.publishConfig === 'object',
```

Comment:
I think you need to check for `registry` as well, as `publishConfig.access: public` is used for scoped packages.

Reply:
Thanks for the heads up!